src/pages/LogDrop/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { useCallback, useRef, useState } from "react";
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import { palette } from "@leafygreen-ui/palette";
+import { Option, Select } from "@leafygreen-ui/select";
+import { Body } from "@leafygreen-ui/typography";
+import { useDropzone } from "react-dropzone";
+import Icon from "components/Icon";
+import { LogTypes } from "constants/enums";
+import { size } from "constants/tokens";
+import { useLogContext } from "context/LogContext";
+
+const { red } = palette;
+interface LogDropProps {
+  onChangeLogType: (logType: LogTypes) => void;
+}
+
+const LogDrop: React.FC<LogDropProps> = ({ onChangeLogType }) => {
+  const { ingestLines } = useLogContext();
+  const [hasDroppedLog, setHasDroppedLog] = useState(false);
+  const [logType, setLogType] = useState(LogTypes.RESMOKE_LOGS);
+  const lineStream = useRef<FileReader["result"]>(null);
+
+  const onDrop = useCallback((acceptedFiles: File[]) => {
+    acceptedFiles.forEach((file: any) => {
+      const reader = new FileReader();
+
+      //   TODO: EVG-17664 replace these with error toasts
+      reader.onabort = () => console.log("file reading was aborted");
+      reader.onerror = () => console.log("file reading has failed");
+      reader.onload = () => {
+        setHasDroppedLog(true);
+        lineStream.current = reader.result;
+      };
+      reader.readAsText(file);
+    });
+  }, []);
+
+  const onParse = useCallback(() => {
+    onChangeLogType(logType);
+    if (typeof lineStream.current === "string") {
+      ingestLines(lineStream.current.split("\n"));
+    }
+  }, [ingestLines, logType, onChangeLogType]);
+
+  const { getRootProps, getInputProps, isDragActive, open } = useDropzone({
+    onDrop,
+    maxFiles: 1,
+  });
+
+  return hasDroppedLog ? (
+    <Container>
+      <StyledSelect
+        label="How would you like to parse this log?"
+        onChange={(value) => setLogType(value as LogTypes)}
+        value={logType}
+      >
+        <Option key="resmoke" value={LogTypes.RESMOKE_LOGS}>
+          Resmoke
+        </Option>
+        <Option key="standard" value={LogTypes.EVERGREEN_TASK_LOGS}>
+          Raw
+        </Option>
+      </StyledSelect>
+      <Button data-cy="process-log-button" onClick={onParse} variant="danger">
+        Process Log
+      </Button>
```

Comment:
It may be helpful to also include a "Cancel" button here in case the user accidentally uploaded the wrong file. They could be taken back to the upload page.
<img width="210" alt="image" src="https://user-images.githubusercontent.com/9298431/189362486-a09d9d1f-0d75-47c9-988a-b13d7339c308.png">